### DISCUSSION TITLE
[ARCHIVED] Augmented Exercise 3 - Object collection

### DISCUSSION THREAD

**Discussion by periode** [periode/augmenting-gallery#12] [Homework]
# Object Collection

Develop an application with ARKit positioning a group of objects around the physical space in order to communicate around a certain theme/idea/concept of a collection. Think about why these objects, what they mean together, what they do to people, and how they contributes to the world. Add text, images, videos, audio in order to enrich your communication, along with some basic interactivity. Following a cultural studies approach, you can start with the three broad inquiries of **production**, **mediation** and **reception**.

Post a screen recording of your app and a link to your Unity project (zipped) below.

**Comment by vb1424** [comment]
Vimeo link to my screen recording: https://vimeo.com/698674889
(It includes a full description for this Exercise.)

Zip file link: https://drive.google.com/file/d/1dkMwcgEipzzQQiQjK7977DWqRLW0xiZ8/view?usp=sharing

Enjoy :)

**Comment by aidanmassie** [comment]
For this assignment, I decided to expand on the previous augmented exercise and build an app centered around buildings that have at one point stood as the world's tallest building. The process was pretty simple, as I just found 3d models online similar to the Empire State Building model I had before, and set them up in a circular pattern so the viewer is able to see something in every direction they look. I wanted to make sure, though, that scale was a priority and that there was a clear size difference between the Chrysler Building and the Burj Khalifa since there is a clear size difference between the two. I think it would have been cool to set it up as a hall of buildings, where they would stay stationary and the phone could move around a physical space to explore the "gallery." Unfortunately, that seemed pretty advanced and time-consuming under the given time constraint I faced. Again, I struggled to get audio working in the app, which was the same problem I had last time. It seems like there's some kind of setting that I can't find on my phone that's preventing the Unity app from playing--I researched for a while again and was still unable to find a solution. Altogether, I'm happy with how it turned out and how the presentation looks!

Screen recording --> https://youtu.be/-lUWp5heF3g

Zip file --> https://drive.google.com/file/d/1I7v3SvaIShojIjGKiXE8Ja6xUZ8FhzUo/view?usp=sharing

**Comment by mdl601** [comment]
Michelle Leidecker 

Here is the link to the zip file: https://drive.google.com/file/d/1JOxGO3vsvdL5scKgo-mu3yQzgxrmA9qe/view?usp=sharing
And the unlisted YouTube video demonstration: https://www.youtube.com/shorts/PdeLEuSQGug

I added more about the history of phones along with coins that disappear via the Raycaster demonstration. Thanks!

**Comment by miafan716** [comment]
![713C231E-9DF0-42B3-90E6-38B2341DC926](https://user-images.githubusercontent.com/55967980/163071767-7ced72ca-8670-4cd4-8ff8-0468a8b9cfe5.jpeg)
I tended to simulate one of the interaction method that Computerspiele Museum applied. Having troubles in going across things in the augmented reality and also don't know how to screenrecord on Android phone (there is no recording icon! and if I download creenrecording software the AR app would be too laggy), so I am clicking with my fingers. The texts were really small in the demo, but sometimes it would be larger, changing according to the camera position. 

My demo: https://drive.google.com/file/d/19MVjn2H8Mphn6_CnvSJSOT4akDWLoXyW/view?usp=sharing
My project zip file: https://drive.google.com/file/d/1Nw3KIl0URm-bVzdUN_dOl1VRB0JxeYK_/view?usp=sharing
Tutorials I've followed
_Raycasting_: https://www.youtube.com/watch?v=ZCrXWAPCMuU&list=PLyQtZRmg0ogd2EDmaPrW6o7XmvTKCis5Q&index=6
_Videolist_: https://www.youtube.com/watch?v=bBRZUrwyPZQ
_Videoplaying_: https://www.youtube.com/watch?v=t-wamt3xilA&t=151s
_Text embedded on 3d object_: https://answers.unity.com/questions/1178633/putting-text-on-a-3d-object.html
_Plane tracking_: https://www.youtube.com/watch?v=VMjZ70PmnPs&list=PL9z3tc0RL6Z4WenhJiJieCcrPVNxYszod&index=4

**Comment by joshblecherman** [comment]
[Here is the link to my video](https://vimeo.com/698824891)
The zip file should be shared with you over google drive.

The music is the "Benet Gesserit" track from Dune (2021). The text is switched by clicking on the still-suit. This is achieved through ray casting. 

I chose to base my collection of objects around the fictional world of Dune. I chose three objects to represent: spice, sand-worms, and still-suits. The relationship between these three objects and what they represent forms the tension at the heart of Dune. Spice is the most valuable commodity in the galaxy, and it can only be found on one planet. One of the largest obstacles to mining of the spice is the sand-worms, but the sand-worms are also the produces of spice. These two facts put the sand-worms in an interesting position when it comes to the production and mediation of this essential substance. Water is the other obstacle to mining spice, and it is represented by the still-suit, a water reclamation suit. On Arrakis, water is a precious resource, while spice permeates the air. The scarcity of a common resource and the abundance of a scarce resource makes Arrakis a truly unique and important place. It demonstrates how important context is when evaluating the value of resources.

**Comment by aashnasoni** [comment]
Link to video: https://youtube.com/shorts/fMOOESh7yHM
Link to drive: https://drive.google.com/drive/folders/1kIZAuwtX5rMXf9tfNsOGSjDkPHPOrSVe?usp=sharing 

There are still quite a few bugs that are not letting me fully execute my collection of objects for this exercise, so I will be updating this link once I troubleshoot the errors and allow a more smooth interface between the objects and text. I want to work further on integrating the audio that doesn't seem to be working along with various text issues that I am facing.

**Comment by zanderzel** [comment]
Link to drive: https://drive.google.com/drive/folders/1t21RrR0bPwJnCjmkfOjYGzO8BcV6Ytv3?usp=sharing

For my project, I've decided to continue with the exploration of music technology and have focused on the different generations of drum machines. The first two photos are Sprites of some of the early drum machines that were commercially produced in extremely small batches: the Rhythmicon (created in 1931 by Leon Theremin) & the Computerhythm (created in 1971 by EKO). The second two objects are 3D models of two ground-breaking machines: The TR-808 (created in 1980 by Roland) & the MPC 2000XL (created in 1997 by Akai/Roger Linn). While it is only partially functioning, my app allows users to tap on the image or model of the drum machine and hear an audio demo of what it sounds like. 

The way in which users cycle to the next drum machine is by tapping on a 'next' button, which would send the user 'through' the image, revealing the next machine. For now, this movement is not functional and we can only hear a demonstration of the first machine. I will be updating the app to complete the whole experience.

**Comment by Lkochar19** [comment]
Collection: Queer Armenian Art
Description: This exhibit aims to bring to light and celebrates notions of “queer” and “other” as they pertain to the Armenian and LGBTQ communities. These are labels and experiences that have been shared by both communities throughout history, serving as sources of oppression, exclusion and pain. While Armenians now have a homeland, the LGBTQ community is constantly navigating within their surrounding cultures. During the last decade, especially in diverse cities such as Los Angeles, the LGBTQ community has been embraced by the larger society.
GameObjects:

- 1 video: LISTEN TO ME - Untold Stories About Hatred
- 1 3D object: Cross (Talk about religion and cultural homophobia)
- 1 audio: Cher
- 1 text
“I encompass in myself the tortured, raped, kidnapped, subjected to physical violence… unemployed, poor and morally abandoned, Armenian transgender’s image.” — Lilit Martirosyan
->first openly transgender person to speak in the Armenian parliament earlier this month
->historic moment by LGBT+ campaigners, but it was also followed death threats.
->she and her colleagues have received numerous death threats
->address has also been leaked online
- 2 images
->Image 1 (book cover): HATE SPEECH DISPLAYED BY STATE OFFICIALS TOWARDS LGBT PEOPLE IN ARMENIA
->Image(s) 2: the Queer photography collection of Maryam Sahinyan

**Comment by fractalessense** [comment]
For this exercise, I was working with an actual industrial-gallery space, which I scanned, and layered over top objects that reflect in different ways Ukraine. Particularly, I am working with personal objects, like the seashell, and photos taken by me in the past of the country at peace. The idea of the project is to AR-map on the digital additions to the actual space, which I haven't quite figured out how to do just yet. The aspects of production, mediation, and consumption (as well as identity and regulation) deal with the realities of collaborative art and sharing personal stories. This space is a currently a Ukrainian Culture Center, and I would like to provide all Ukrainians with the opportunity to share their personal content of collaboratively compose it to tell fresh stories. The question of mediation and regulation is an important one with collaborative art, though in mind mind, I view the process as merely an opportunity for further collaboration. The consumption of this piece would be providing a space for people to reflect on events in a setting/tone that is not so much didactic, as a personal sharing of story. I look forward to all the branches that this project can evolve in!

https://drive.google.com/drive/folders/15A465S_218KsklVxQdoLLnJtof4yHdA5?usp=sharing

**Comment by yylinj** [comment]
Drive Link for the file zip: https://drive.google.com/file/d/1R-jgbOR8im50VEC10tCE1GSm7P2_kp6y/view?usp=sharing

For this assignment, I explored objects and things that remind me of home. From the fruits and veggies from the garden to subway stop announcements to birthday cakes and home videos. My texts go through the stories of each one of these and why they are memorable objects that I wish to put in this collection.

**Reply by yylinj** [reply]
HI, here's the link to my video. The sound didn't get recorded. https://drive.google.com/file/d/1-e75g1w0fdr_shyPkXvhlGb445qN40QM/view?usp=sharing

**Comment by Mg5970** [comment]
For my assignment I chose to use Tesla cars as they are an intersection between production mediation and reception especially in an age of climate change activism and social media activism with a figure such as Elon Musk at the helm of the company. I created a Tesla show room with three different Tesla vehicles, four plants, a table, and a Tesla bot which gives information on Tesla when you ring the bell at its desk at which point text is cycled through. I do not want to say too much more as much information is presented in the text cycling.

Link for the file zip and video:
https://drive.google.com/drive/folders/1o5wmmrAcohOwrR5hS3b3pi7tzrSR-0hP?usp=sharing

**Comment by luyumenq** [comment]
I decided to model the classroom and add interactive GameObjects inside the space. Some iconic objects I picked are the Projector, the Heater, the Microphone and a coffee kettle. When the user fixes the center of the screen to one of the above objects, a button will pop up and inside it is the information about the object. 

What I managed to incorporate:
- Raycasting feature that detects if the user is pointing at an object
- The interaction between the raycast and the object colliders

Some features currently not available yet with my skill: 
- The area of the recast is too small (just one line). The user has to point the device to the object SUPER specifically. This could be solved by  increasing collider sizes, but I think it may cause more confusion to Unity.
- I initially tried to make use of the classroom itself and put empty GameObjects in it. However, I don't know how to track the player's specific position in the room.  

https://user-images.githubusercontent.com/72938590/202030638-11e6aa9d-f41e-4311-a7a4-a2ea81588af1.mov

**Comment by izzyrieken** [comment]
I worked with some 3D scans I took on my vacation and was interested in creating a sort of 3D collage that maybe I could develop into a larger project later.

**Reply by izzyrieken** [reply]
https://drive.google.com/file/d/1b5Q4t060wVp0KS28Y6unBMmcmuCDfNDr/view?usp=drivesdk

**Comment by Dav1dyang** [comment]
I converted the previous unity assignment into an Art with Xcode and Reality Composer.

There are few things I want to implement for a better user intentions, however, I don’t have much experience with Xcode so some of the things I’m trying to do is limited by the my understanding of technology.

here’s a link to the video: https://youtu.be/ZkBkkOexeCQ

**Comment by kylewenchang** [comment]
I continued with my same object from the first Object Story project, the water bottle. Most of the information is the same but I adapted the presentation of the assets to be better suited for AR. A small onboarding screen tells the user what to do to begin the experience, and I used raycasting and plane tracking in order to place objects on a physical surface. Rather than having the navigation on the same canvas that the object information was displayed on, I decided to try a "remote-style" navigation UI with the buttons being static overlays on the screen. This was a little bit of a challenge as I used separate scripts for Remote Management and Slide Management. I ended up having to share the "current page" variable across scripts, and using that value to access a specific child of the newly instantiated prefab with all 3 slides, enabling or disabling the appropriate slide when necessary. I also used a script to make the object information plate always facing the user, but perpendicular with the table. For the last slide, I added some physics to a bunch of water bottle models, and made them collide with detected planes in order to emphasize the unseen/unfelt effects of the mass consumption/convenience of bottled water. Some things I would have liked to add are object occlusion (I'm not sure about Unity AR's support for this) to prevent water bottles that should be on the floor being visible through the table, shadows that overlay over surfaces to make the AR objects feel more physical, and collision sounds give the falling/bouncing bottles some more physicality as well. I also removed some of the photographs from the original project, only opting for the models because I felt that the photos also led to a little bit of a disconnect with physicality to me, but I'm not sure where to draw the line between immersion and information.

[Video Demo Here](https://www.youtube.com/watch?v=Tqc6IfS89Ps)
[Files Here](https://drive.google.com/file/d/1hVB1a-Se1R87aVEzgt4iiSdTb025u7QC/view?usp=sharing)

**Comment by IanPort** [comment]
I decided to utilize my previous project to clean up, restructure, and experiment with another XR capability. I moved the four main buttons (Pandemic, Epidemic, Necessity, and Pleasure) around the AR camera so the user would be the center link between the concepts. In my first attempt at the app, 2 of these concepts/buttons/displays were not visible, but now each is accessible. Other improvements include using images as textures for the wall texts (changing the blank backgrounds), changing button texts for legibility, and adding an audio component to the experience (insulin pump soundscape + testimony). 

I replicated the floating syringe and vial, placing both in a diagonal that guides the viewer. One has instructions to “look around” and the other to “scan your own vial.” While it’s a bit finicky, I tried to use the Reference Image Tracking to give any viewer with an injectable medication a special moment of acknowledgment. A bandaid is the object that appears using this function. The ideal would be a counting system, in which the bandaid would reveal the number of people who have scanned their vials prior to the current user. Whether it be insulin or hormones, the select people who depend on syringes could form a symbolic measure of community/identity.  

[Google drive (with video and zip file)](https://drive.google.com/drive/folders/1rymeJaPY2n9Vz6MnAIVTXLk8-PU-7suq?usp=sharing)

**Comment by kenparks12** [comment]
For my 3rd exercise, I wanted to extend the living room space that I have been using for my PlayStation exhibit. The user is able to walk through the TV, literally into the "PlayStation universe". In the exhibit, there is a collections of related items and concepts surrounding the PlayStation 2's legacy, including photos, 3d models, video, audio, and text. 

[Drive with video and ZIP](https://drive.google.com/drive/folders/1nty6pxFCrBmT-8S_wzPXLv05U4-P-wCp?usp=sharing)

**Comment by mariaabzm** [comment]
For my 3rd exercise, I took my 2nd project and rearranged my objects to surround the user. I wanted to capture the idea that you're always being watched and different ways of seeing through the experience.
[link to zip + video](https://drive.google.com/drive/folders/1EpMNrft5EA_2NQZoC5VTKwF3b4k1m22B?usp=share_link)

**Comment by olivestew222** [comment]
For my third exercise I explore Gustav Holst's Composition: _The Planets_. I arranged the planets into the space so that users could view and interact with them. They give information about their corresponding part in the suite, which plays when beginning the AR experience. I added clouds and asteroids to blend into the scene as well, but not be too overpowering. 

link to file/vid = https://drive.google.com/drive/u/1/folders/1WePvF3v8f6ihlqOYUElP_Lb5vxBd4W22

**Comment by hannah-y-zhao** [comment]
For this exercise, I decided to focus on the relevance and concrete applications of currency. While a lot more obvious than envisioning any potential future currency, my intention was to emphasize how money was a connection between practically any material (and often nonmaterial) thing.
All my 3D models came from the asset store, and I tried to add citations for my quotes but unfortunately it doesn't show up in AR. I enjoyed being able to use colliders for invisible triggers and I got more comfortable with parent/child relationships.
[Video demo](https://youtu.be/0Vn887Sioy0)
[Zipped files](https://drive.google.com/drive/folders/1cQux9ycD27d0g21J3cbaads0lWQjNroE?usp=sharing)

**Reply by periode** [reply]
@hannah-y-zhao 

grade: A

Your spatialization of the app is great, and provides both a sense of mystery and of solemnity, hinting at the special status of money in our societies, and leading to a big reveal at the end. Here are a couple of thoughts on how you might have improved the app:

- Showing a screenspace UI at the very beginning to let users know how to interact (by walking)
- Considering a more linear reveal by having pedestals appear one after the other, after each previous pedestal has been inspected
- As a further step, you might want to think about how to follow-up on "All about currency"; how do you transition into a more traditional knowledge delivery? For instance, you could have a ScrollRect UI element in Screen Space that introduces some of the basic facts and questions about money, or links towards other websites introducing it.

**Comment by Yupu-Chen** [comment]
[Screen recording of the application](https://drive.google.com/file/d/1poVy9mZFzgLNBMiAuAUjzX2Dpz53OXZG/view?usp=sharing)

[Zip file](https://drive.google.com/file/d/17Xoi6hog1SzpIXsgdN6bv28A3Qfzb1oM/view?usp=sharing)

For this project, I keep the theme of flight. I attempted to explore the relations of flight, technology, and environment. There are four sections in this application, The user could always use the buttons at the bottom to change sections. I included colliders, UI buttons, and interactable objects in the interactivity aspect of this project. Contentwise, I explored the development of flights, inside and outside flight, and low-carbon-footprint ways of flying, trying to talk about flights as a cultural object from production, mediation, consumption, and regulation.

Hope you enjoy ;)

**Reply by periode** [reply]
@Yupu-Chen 

Grade: A

You make a very good use of AR to allow the user to explore different models of planes, and the different parts. However, the way they are displayed, and the UI layout with each object can be confusing at times. Here are some thoughts which could improve it:

- Only show one object at a time, with the corresponding text title. For instance, when you show both the seath and the engine, when user clicks on the seat, make the engine disappear (it could be displayed again with another button press), and add a first line saying "SEAT" (or something along these lines) in the text, in order to make it clear what this text is referring to.
- The UI elements are laid out in very different ways: some at the bottom (for the next/previous), some parented to objects (seat/engine) and some on the side (qatar airways/air china). I would separate the buttons more clearly: system buttons (previous/next) could be in a different size/color or even arrow icons; while object buttons (seat/qatar/etc.) should all be positioned in the same consistent way (either screen or world space), also with a different size/color.

Finally, you could harness the affordances of 3D models by trying to use scale as a means of communication: by putting things at scale (e.g. the Boeing vs. the Wright Flyer, or the difference between the engine size and the seat size) you can communiate a lot about what has been achieved technologically.

**Comment by emersonmack** [comment]
[here](https://drive.google.com/file/d/13XZ9-cWxLS4vT9owL2Qg5czCeCHIQmh1/view?usp=share_link) is the zip file for my project and [here](https://drive.google.com/file/d/14fTwZgUwLVJKXrD1XHqOmMaR4XKX6QtD/view?usp=share_link) is the video. Due to technical difficulties, this video does not show the box trigger features that are scripted into the project. The plan is to have text appear above each planet as the camera approaches it to show information about each one. Revisions will be made for the visual to match the commands. My gallery is simply an exhibit of super cool planets that I find interesting and worth sharing. Viewers are able to orbit, touch, and read about each planet as they float through space. 
In this project, I was much more comfortable with the organization of objects and text in their position and scale. Coding was difficult and I am still having some trouble with operations but the functions are planned and written. I enjoyed experimenting with this project.

**Reply by periode** [reply]
@emersonmack 

Grade: A

Your epxloration on the planets is a very nice use of AR, and is a nice twist on the traditional "check this Solar system", and made me learn some new things! Here are some ways you could develop this further:

- The UI Text is still a bit hard to read: you might consider adding a background image on each of the planets' texts, and make your app title "Super Cool Planets" bigger and at the top, so that it doesn't prevent from reading or interacting with the planets.
- To make it a bit more compellig, you could use the function "Transform.LookAt()" in order to make the planet's text rotate automatically towards the camera and provide a better reading experience.
- In terms of content, consider what are the broad categories why those planets are cool: density? texture? size? Highlighting those with tags/category names would help the visitor grasp the information more easily.

It would also be interesting to consider what is the next step, in terms of information: what questions can you prompt the visitor with? what are the "super cool"/fascinating new developments in space exploration? For instance, mentionning  the James Webb telescope might be one direction

**Comment by aacarcrash** [comment]
[YouTube Link](https://youtu.be/skQSQ4A-jLg)
[Drive Link](https://drive.google.com/file/d/1QarpcibK2l7agePOOJx0jMaTZSI9wAyZ/view?usp=share_link)

The project is an exploration of the Indian internet culture in 2000s and early 2010s. The net landscape was shaped by cyber cafes which is where this exhibit is set. The objects are a collection of websites and digital artefacts that I believe are crucial to the identity of the Indian internet in that era. It was a challenge getting the AR camera collision to work but I realized what the issue was. My project initially didn't have a floor so whenever I added rigidbody to the camera, it just fell down to infinity. Adding a floor fixed the issue but I would like to investigate how to work around this without adding a floor, since that might not creatively work with some projects.

**Reply by periode** [reply]
@AakSin 

Grade: A+.

The layout and scenography is very impressive and relevant to give an immersive feeling of being in an internet cafe! Similarly, the screen space UI's colors and fonts make it very diegetic. To this, I would add the name of the piece/website currently being looked at by the visitor in the title bar of the popup window.

One suggestion I would have, is whether it is necessary to have the introduction panel in World Space: it might be more readable to start in Screen Space, and then have a button at the bottom that says "enter the cybercafé" which then instantiates all the 3D models.

**Comment by BerryBomarB14** [comment]
Using ARKit to position a group of objects around physical space is a great way to communicate a theme or idea. It's important to think deeply about why you chose these objects and how they contribute to the world. Adding text, images, videos, and audio will definitely enrich the experience and make it more interactive.

**Reply by periode** [reply]
@BerryBomarB14  are you a bot?

**Reply by periode** [reply]
@BerryBomarB14 

Grade: F

Very sporadic engagement.

**Comment by emersonmack** [comment]
[Here](https://drive.google.com/file/d/1o35NOY_57psKllnIRj6ODoUV0ALPbv6c/view?usp=share_link) is my finished project file and [here](https://drive.google.com/file/d/1nQx6k1NKwmaDIKP6U7-oUrXM-YryEGCk/view?usp=share_link) is the video. I managed to rework my box colliders to trigger the text above each planet when the camera was in the zone around it. The camera is placed in the middle to allow the space between each planet to not be too far in either direction for the viewer. I felt more comfortable writing scripts and organizing my thought in this project and I am happy with how well it turned out. Next time, I would like to experiment more with audio and animation. Orbiting planets could be a cool revision to this project.

**Comment by Opal42** [comment]
Files here: https://drive.google.com/drive/folders/113UIu14VCheyny6pyD_ZI-jVPfBAQnt3?usp=sharing

The Roundabout Museum was designed largely around me trying to figure out how to work around the many technical issues my Android setup has been having with movement in AR space. To those ends, I designed this experience to be a more linear one, where revolving around the space brings you through its narrative, building up layers of information, going around and around, just as the process of building narratives around art objects itself does too. I wish that there was more that was done with the audio at the end, but I struggled to find sfx and bgm transitions that felt appropriate, and decided that less was more in this case.

**Reply by Opal42** [reply]
Also, I spent a ton of time trying to set up a system that would reference multiple triggers around the circle, so that you could go in either direction, and that the sequence would progress after all the triggers were activated. But no matter how I tried, I couldn't get global variables to work properly in Unity. (It's now just one trigger at the end of the clockwise direction, which does allow for a problematic sequence-break if going the other way.) Would appreciate a hand looking at that later on.

**Reply by periode** [reply]
@Opal42 

Grade: A+

Very impressive interaction design and content curation! You make a very good use of the spatial layout of objects in relation, and seamlessly suggest deep questions about artefact, stories and histories.

Here are a couple of thoughts on the user experience. Since it's not a means of interaction that users are very used to, you could use more UI techniques to convey what is going on. For instance:

- have an arrow on the first panel to show in which direction to turn
- have a number of turns completed/turns to complete on the first panel to give a sense of progression to the user
- show which are the newer vs. older panels on each object by changing slightly the opacity/color (newer panels stand out more than the previous ones)
- show which panels have been changed once tapped on. the audio effect is good, but reinforcing it with a visual effect (e.g. picking a random blue tint) could reinforce it very well.

**Comment by daniaezz** [comment]
[link to the drive and video](https://drive.google.com/drive/folders/1FwMqDaaP3JWo98v-qD7i8L5AhZH9gLeO?usp=sharing)

For this assignment, I decided to create a gallery of the different Nintendo consoles over time. I had the objects arranged in a zigzag pattern so that it does not feel like a simple straight timeline because I believe there is a lot of overlap when it comes to Nintendo’s timeline. I also decided to add the dates when the consoles were supported so that it feels more concrete than just having the objects lying around in case the user wanted to deviate from the intended path. Originally, I wanted to have the consoles with screen play a video when the user came closer and press buttons on the console itself to change the video. However, because of the nature of the 3D models, I decided to scrap the idea of the buttons as they did not look clickable but instead looked like part of the 3D model. After adding videos to all the consoles with the screens, I felt like the videos were distracting from the texts so I decided to limit the videos to just 3 consoles instead.

**Reply by periode** [reply]
@daniaezz 

Grade: A

The spatial layout of the objects, their accompanying texts and videogames displaying on the screens of the handhelds makes for a very compelling experience! On the presentation note, it could have been nice to connect the home consoles to a TV monitor in order to show the most symbolic games that were played on it.

On the down side, there isn't so much context provided to your collection, so it feels like a historical display of the most Nintendo devices, but without being exhaustive or too questioning. What about the Virtual Boy? What about the Nintendo playing cards? As players of these consoles, it might be easy for us to rely on nostalgia, but we have to keep in mind those who've never played them: why should they care about video game consoles? Why should
they care about Nintendo more than, say, Sony or Xbox? For instance, focusing on the company's philosophy, on the "Nintendo Seal of Approval" or on the focus on casual gaming vs. hardcore gaming could be some starting points.

**Comment by cz2109** [comment]
Link to Google Drive and Video: https://drive.google.com/drive/folders/1Q_2dv7qUMQIBAeyE0KO0tPtErhL3zZqL?usp=sharing

For my 3rd exercise, I decided to research iconic chair designs and create a collection of 6. I oriented them on a floor and coded scripts that allowed the first person to trigger a collider to show text for each object. Delays in submission were caused by a build error, remedied by deleting the library folder within my project, as well as computer technical difficulties, remedied by trying 5 different computers (lol). Overall, I found this exercise to be my favorite, as it combined elements of the first two, but allowed me to assess my progress and confidence with the software and adopt a greater breadth of skills (e.g. understanding the function of colliders, screen space navigation, increased comfortability with UI elements, etc.). I also really enjoyed researching this topic and streamlining my exploration into AR.

**Reply by periode** [reply]
@cz2109 

Grade: A


Your collection of chairs shows a wide overview of something that is a near-universal fixture of human societies across time and space. It's a very nice group of samples, but I would encourage you to push a little bit further in terms of questions being asked: what's interesting about chairs? Are they very hard or very easy to make? Are they essentially all the same or all different? What is interesting about their construction methods?

Additionally, one way you could develop this app further would be to develop further the spatial aspect:

- Making sure all models are at scale 1:1
- Only displaying the chair models on the floor around the user, rather than on a virtual floor,
- Adapting the text font / color of the background image to the style and era of the chair.

**Comment by mariamabdelrazekk** [comment]
Video and Drive link: https://drive.google.com/drive/folders/1Zo9-sRjiZSKOTnB5rRNcM6_ZpDghSDTf?usp=share_link

My collection revolves around a group of artifacts from different areas around the world, non of which are British, all displayed in the British Museum. The aim of this project is to show that many of these artifacts, do on in-fact belong in the British Museum. I was hoping this project could have an element of "funny" in it, however, I did not know how to include it, I was thinking of using ironic music, but could not find tracks that really made sense with the theme. I faced multiple problems, starting off with the scale, you can see how large the objects are through the text and how much I had to walk to show everything. I minimized  all my assets as I put them in an Empty object which I rescaled to 0.2. In my next project, I will put the scale into consideration from the start to avoid this. I also had issues placing all of the objects in the same line of sight and walking as me, so I had to hold my phone with an angle facing upwards most of the time, which I think would be uncomfortable on the long run. I added spot lights to every object that turned on once the rigid body hit the assigned collider, however, I do not think they are very clear as I filmed this in the day. I did learn a lot working on this project; working with global variables being the highlight.

**Reply by periode** [reply]
@mariamabdelrazekk 

Grade: A-


It is a very good idea to convert your exhbition about the roestta stone from a learning experience to a critical commentary about the acquisition policy of the British Museum (to say it politely). Bringing all these artefacts together provide a spin on what the museum is usually housing (this collection), with the tongue in cheek reference to the "British section".

However, as you've mentioned, there are a couple of UX issues that prevent the visitor from getting the full experience. Here would be some suggestions for improvement:

- The text accompanying every piece is very informative and dry, as would be displayed in the BM. Perhaps you could have focused on where the piece is from, when it was taken by the BM and by whom, and whether anyone is asking for restitution.
- Each of the objects could have a floating text indicating from which country they are (e.g. "Egyptian section", or "Ethiopian Section")
- The final section should be more explicit: there are no objects in the British section because the BM is mostly about foreign objects. You could then have a few text panels introducing some of the debates and recent changes in terms of spoliation, restitution, etc.
